pxr/usd/pcp/cache.cpp

```diff
@@ -1229,7 +1229,7 @@ PcpCache::ReloadReferences(PcpChanges* changes, const SdfPath& primPath)
     // local layers.
     SdfLayerHandleSet layersToReload;
     for (const PcpLayerStackPtr& layerStack: layerStacksAtOrUnderPrim) {
-        for (const SdfLayerHandle& layer: layerStack->GetLayers()) {
+        for (const SdfLayerHandle layer: layerStack->GetLayers()) {
```

Comment:
Just a heads up this has been fixed internally with https://github.com/PixarAnimationStudios/OpenUSD/pull/2643.  You might need to rebase after the next push.